File: src/main/java/net/hectus/neobb/turn/default_game/block/TPackedIce.java
package net.hectus.neobb.turn.default_game.block;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.turn.default_game.attributes.clazz.ColdClazz;
import net.hectus.neobb.turn.default_game.attributes.function.AttackFunction;
import org.bukkit.block.Block;

public class TPackedIce extends BlockTurn implements AttackFunction, ColdClazz {
    public TPackedIce(NeoPlayer player) { super(player); }
    public TPackedIce(Block data, NeoPlayer player) { super(data, player); }

    @Override
    public int cost() {
        return 3;
    }
}

File: src/main/kotlin/net/hectus/neobb/modes/turn/default_game/block/TPackedIce.kt
package net.hectus.neobb.modes.turn.default_game.block

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.modes.turn.default_game.attribute.clazz.ColdClazz
import net.hectus.neobb.modes.turn.default_game.attribute.function.AttackFunction
import net.hectus.neobb.player.NeoPlayer
import org.bukkit.block.Block

class TPackedIce(data: Block?, cord: Cord?, player: NeoPlayer?) : BlockTurn(data, cord, player), AttackFunction, ColdClazz {
    override val cost: Int = 3
}
